tools/run_tests/run_xds_tests.py

```diff
@@ -109,6 +110,10 @@ def parse_port_range(port_arg):
     default='',
     help='File to reference via GRPC_XDS_BOOTSTRAP. Disables built-in '
     'bootstrap generation')
+argp.add_argument(
+    '--xds_v3_support',
+    default=False,
+    help='Support xDS v3 via GRPC_XDS_EXPERIMENTAL_V3_SUPPORT')
```

Comment:
Should this say "Support xDS v3 via GRPC_XDS_EXPERIMENTAL_V3_SUPPORT. If a pre-created bootstrap file is provided via the --bootstrap_file parameter, it should include xds_v3 in its server_features field."?

Since it looks like this also requires putting `xds_v3` in the bootstrap file `server_features`, which is done here when we generate the bootstrap file, but it could be overlooked when using a separately created bootstrap file (which is used in staging).

Reply:
Makes sense. Done.